packages/storefront-sdk/src/client/index.ts

```diff
@@ -1,15 +1,77 @@
 import { createClient } from '@urql/core';
 import type { Client as UrqlClient } from '@urql/core';
 import type { StorefrontClientParams } from '../index.js';
+import type {
+	SetConfigParams,
+	SetConfigResponse,
+	StorefrontConfig
+} from '../types/config.js';
 
 export class StorefrontClient {
-	readonly #graphqlClient: UrqlClient;
+	#graphqlClient: UrqlClient;
+	#storefrontEndpoint: string;
+	#previewToken: string | undefined;
+	#currencyCode: string | undefined;
```

Comment:
I think we can remove `#currencyCode` altogether, because we discovered that it doesn't have any purpose in `@nacelle/storefront-sdk@1.x`.

```suggestion
```

Reply:
NOTE: this will also require changes in test files and `src/index.ts`.